### PR TITLE
Show the right request/response in repos/comments.md

### PR DESCRIPTION
**Pull request by Shredder121**

### CODE REVIEW

**Comment by keavy** [comment]
Thanks @Shredder121, it looks like the body differs in the response of the create example also, would you like to do the same there?

**Comment by Shredder121** [comment]
You're right! I fixed that now. :sparkles:

File: content/v3/repos/comments.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Name | Type | Description
 ### Response
 
 <%= headers 200 %>
-<%= json :commit_comment %>
+<%= json(:commit_comment) { |h| h['body'] = 'Nice change'; h } %>
```
Comment (Shredder121) [inline content/v3/repos/comments.md]: I can also make it a little lighter like this:
```
<%= json(:commit_comment) { |h| h.merge('body' => 'Nice change') } %>
```
What do you prefer?

Comment (keavy) [inline content/v3/repos/comments.md]: Oh good call, yes let's go with `h.merge('body' => ...`

Comment (Shredder121) [inline content/v3/repos/comments.md]: I just updated the pull request.

**Comment by Shredder121** [comment]
Of course I am being rate limited.

**Comment by keavy** [comment]
Maybe missing a push? For the same on the create example?

**Comment by Shredder121** [comment]
I'll just repush it.

**Comment by Shredder121** [comment]
And again.

**Comment by Shredder121** [comment]
Cool cool, off hours.

**Comment by keavy** [comment]
Thanks! :zap:

**Pull request merged**